nutanix/resource_nutanix_image.go

```diff
@@ -139,6 +139,26 @@ func resourceNutanixImage() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"data_source_reference": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Computed: true,
```

Comment:
If the field is marked as "Computed" then we should ideally read its value in resourceNutanixImageRead().
It calls out to https://www.nutanix.dev/api_references/prism-central-v3/#/b3A6MjU1ODc1MTE-get-a-existing-image API. 
We can set this value in read function as well.